r/tests/testthat/test-python.R

```diff
@@ -24,6 +24,8 @@ test_that("install_pyarrow", {
   skip_if_not_installed("reticulate")
   # PyArrow doesn't support Python 3.6 or earlier
   skip_on_python_older_than("3.7")
+  # no pyarrow wheels for macos 10.13
+  skip_if(Sys.info()["sysname"] == "Darwin" && Sys.info()["release"] < 14)
```

Comment:
IIUC the release here should actually be 18 (cf https://en.wikipedia.org/wiki/Darwin_(operating_system))

Reply:
oh wow that versioning scheme is confusing :D